delegation-verify-coordinator/templates/deployment.yaml

```diff
@@ -41,11 +41,14 @@ spec:
             {{- end }}
           command: ["/bin/sh", "-c"]
           args:
-            - "invoke create-database || true; invoke init-database --mins-ago 20"
+            - "invoke create-database || true; invoke init-database"
```

Comment:
I think putting some logic here is not really elegant and it lacks some errors handling.
We assume that if `create-database` fails, then it means that the db is already created, and so we just need to init the database.
What happens if we want to create the database but for some reasons it fails creating it?
We need to run/call a small script that handle errors and provide logs for debugging

Reply:
Yes agreed. I will come up with a entrypoint script that we mount as a configmap and checks for possible errors and handles them appropriately.